install_everything.sh

```diff
@@ -108,7 +108,7 @@ if ! [ -d $NUMAP_INSTALL_ROOT ]; then
     echo "Installing numap..."
     rm -rf $NUMAP_ROOT || exit 1
 
-    git clone https://github.com/numap-library/numap.git $NUMAP_ROOT || exit 1
+    git clone https://github.com/jklinkenberg/numap.git $NUMAP_ROOT || exit 1
```

Comment:
Why change the Numap repo ? Are there changes in your repo that are needed and can't be merged into the "official" numap repo ?

Reply:
Also related to my other comment. Can be ignored.